src/bci_build/package/mariadb/README.server.md.j2

```diff
@@ -0,0 +1,66 @@
+# {{ image.pretty_name }} Container Image
+
+MariaDB is one of the most popular database servers in the world. It's made by the original developers of MySQL and guaranteed to stay open source.
+
+## How to use this image
+
+By default, the image launches MariaDB with the same configuration as the one
+that comes with SUSE Linux Enterprise Server. The differences are that logging is
+sent to stdout and binding to `localhost` is disabled.
+
+The only environment variable that is required to start the container is the MariaDB root password.
+
+```ShellSession
+$ podman run -it --rm -p 3306:3306 -e MARIADB_ROOT_PASSWORD=my-password registry.suse.com/suse/{{ image.name }}:{% if image.is_opensuse %}latest{% else %}{{ image.version }}{% endif %}
+```
+
+or:
+
+```ShellSession
+$ podman run -it --rm -p 3306:3306 -e MARIADB_ALLOW_EMPTY_ROOT_PASSWORD=1 registry.suse.com/suse/{{ image.name }}:{% if image.is_opensuse %}latest{% else %}{{ image.version }}{% endif %}
+```
+
+## Volumes
+
+The default data directory location is `/var/lib/mysql`.
```

Comment:
it's not the default, it's the only supported location afaik

Reply:
I'm not sure, I believe if you change the settings it might be possible to use another location. I'll see if I can come up with a better sentence.

Reply:
So we support changing the locations and we're sure that it will never be hardcoded anywhere? I think if noone requested that feature we just don't advertise it for now because it's just more work to test and not provide any additional value IMHO

Reply:
I updated the message here to imply that this is the only supported location.